### PR TITLE
fixed the bug by adding a cleanup function to the useEffect in LoginC…

### PR DESCRIPTION
**Pull request by AustinGreer**
…ontainer

## Description

Clicking the back button when on the register webpage errors the entire webpage and the webpage turns white. This was due to the LoginContainer component not having a cleanup function in the useEffect hook. Therefore, I added the remove method from Okta widget to run in the cleanup function.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ const LoginContainer = () => {
         throw err;
       }
     );
+
+    return () => {
+      widget.remove();
+    };
```
Comment (bukit3point0) [inline src/components/pages/Login/LoginContainer.js:54]: Not much to say! Easy fix, good work.

**Review by bukit3point0 — APPROVED** [review]

**Review by Eddie98Lopez — COMMENTED** [review]
Noice work. Seems legit.

**Comment by salvare736** [comment]
Good catch.

**Review by salvare736 — APPROVED** [review]
Sometimes a simple change is all you need

**Pull request merged**